main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 		cli.StringFlag{
 			Name:   "token",
 			Usage:  "Drone API token from your user settings",
-			EnvVar: "DOWNSTREAM_TOKEN,PLUGIN_TOKEN",
+			EnvVar: "DOWNSTREAM_TOKEN,PLUGIN_TOKEN,DRONE_TOKEN",
```

Comment:
This should be the first item, not the last. After that we can merge that.

Reply:
Fixed